tcwebhooks-core/src/main/java/webhook/teamcity/BuildStateEnum.java

```diff
@@ -14,8 +14,9 @@
 	BUILD_BROKEN			("buildBroken", 			"broken",								"Build Broken",						"Finished (Broken)"),
 	RESPONSIBILITY_CHANGED	("responsibilityChanged",	"changed responsibility",				"Build Responsibility Changed", 	"Responsibility Changed"),
 	BUILD_PINNED			("buildPinned", 			"been pinned",							"Build Pinned",						"Pinned"), 
-	BUILD_UNPINNED			("buildUnpinned",			"been unpinned",						"Build Unpinned",					"Unpinned");
-    
+	BUILD_UNPINNED			("buildUnpinned",			"been unpinned",						"Build Unpinned",					"Unpinned"),
+	BUILD_TAGGED			("buildPromotionTagsChanged","been tagged",							"Build Tagged",						"Tagged");

```

Comment:
```suggestion
	BUILD_TAGGED			("buildTagsChanged","been tagged",							"Build Tagged",						"Tagged");

```
The fact that tags are for a BuildPromotion are an implementation detail I don't think we need to concern ourselves with in this code. We can just use a name that fit in with the other names.